pkg/pillar/types/assignableadapters.go

```diff
@@ -200,8 +221,13 @@ const (
 	IoNetWLAN IoType = 5
 	IoNetWWAN IoType = 6
 	IoHDMI    IoType = 7
-	IoNVME    IoType = 255
-	IoOther   IoType = 255
+	// enum 8 is reserved for backward compatibility with controller API
+	PhyIoNVMEStorage IoType = 9
+	PhyIoSATAStorage IoType = 10
```

Comment:
Why aren't these constants using the identical prefix as the current constants? "Io" and nothing else?
I'm inclined to merge this and fix that later.